runtime/inspector.rs

```diff
@@ -59,12 +58,12 @@ impl InspectorServer {
     let (shutdown_server_tx, shutdown_server_rx) = oneshot::channel();
 
     let thread_handle = thread::spawn(move || {
-      crate::tokio_util::run_basic(server(
-        host,
-        register_inspector_rx,
-        shutdown_server_rx,
-        name,
-      ))
+      let mut rt = crate::tokio_util::create_basic_runtime();
+      let local = tokio::task::LocalSet::new();
+      local.block_on(
+        &mut rt,
+        server(host, register_inspector_rx, shutdown_server_rx, name),
+      )
```

Comment:
Should there be a `run_local` function in  `tokio_util` that sets up a Runtime with a `LocalSet`?

Reply:
It's not required anywhere besides inspector, so I suggest leaving it as is